packages/components/src/Progress/Progress.scss

```diff
@@ -40,7 +40,7 @@ $gradient-part-width: 100% / 6;
 		&.infinite {
 			background: transparent;
 			width: 100%;
-			animation: infinite-progression-keyframes 5s infinite;
+			animation: infinite-progression-keyframes 2s infinite;
```

Comment:
I'd like to add that you're not increasing the progress bar speed, but the time it took to the bare to achieve a complete tour.

The speed will depends of the progress bar width. Maybe this value should be part of the progress bar API ?